sysutils/sshpass/Portfile

```
@@ -30,6 +30,6 @@ long_description    SSH's (secure shell) most common authentication mode is \
 homepage            http://sourceforge.net/projects/sshpass/
```

We should change all URLs to HTTPS while we're at it:

```suggestion
homepage            https://sourceforge.net/projects/sshpass/
```